src/style.css

```
@@ -0,0 +1,42 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 :nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+}

```

```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
.stars--3 :nth-child(-n + 3),
.stars--4 :nth-child(-n + 4),
.stars--5 :nth-child(-n + 5) {
  background-image: url("images/star-active.svg");
}

```
Це все можна об'єднати